lib/cli.ex

```diff
@@ -38,6 +38,15 @@ defmodule Onigumo.CLI do
   end
 
   defp usage_message() do
+    IO.puts("""
+    onigumo: invalid usage
+    Usage: onigumo [OPTION]... [COMPONENT]
```

Comment:
```suggestion
    Usage: onigumo [OPTION]… [COMPONENT]
```

What about Unicode elipsis?